krm-functions/lib/kptrl/kptrl_test.go

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I guess you would be adding tests later .

Reply:
done. tests added with 100% coverage